### ISSUE TITLE
Axis2.xml userAgent and server values 

### ISSUE THREAD

**Issue by AmaliMatharaarachchi** [wso2/product-apim#5503]
**Description**
Axis2.xml userAgent and server values are 2.1.0 in other product versions.
```
<parameter name="userAgent" locked="true">WSO2 AM 2.1.0</parameter>
<parameter name="server" locked="true">WSO2 AM 2.1.0</parameter>
```

**Affected version**
2.6.0
2.2.0

**Comment by bhathiya** [comment]
@AmaliMatharaarachchi can we fix this in 3.0 and close the issue please?

**Comment by tharindu1st** [comment]
This is fixed in latest release
